roles/servicetelemetry/tasks/component_prometheus.yml

```diff
@@ -9,6 +9,7 @@
 
 - name: Create an instance of Prometheus
   k8s:
+    merge_type: merge
```

Comment:
Will need to re-test this. I was working through some issues but I don't think this is necessary.